concert/cloudformation.yml

```diff
@@ -383,6 +386,65 @@ Resources:
         - DynamoDBCrudPolicy:
             TableName: !Ref Table
 
+  AggregateChoiceDataStream:
+    Type: AWS::KinesisFirehose::DeliveryStream
+    Properties:
+      DeliveryStreamName: !Sub ${Environment}-enchanted-brain-aggregate
+      S3DestinationConfiguration:
+        BucketARN: !GetAtt AggregateChoiceDataS3Bucket.Arn
+        BufferingHints:
+          IntervalInSeconds: 60
+          SizeInMBs: 50
+        CompressionFormat: UNCOMPRESSED
+        RoleARN: !GetAtt AggregateChoiceDataStreamRole.Arn
+        CloudWatchLoggingOptions:
+          Enabled: true
+          LogGroupName: !Ref AggregateChoiceDataStreamLogGroup
+          LogStreamName: !Ref AggregateChoiceDataStreamLogStream
+
+  AggregateChoiceDataStreamLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub ${Environment}-enchanted-brain-aggregateChoiceDataStreamLogs
+      RetentionInDays: 30
+  AggregateChoiceDataStreamLogStream:
+    Type: AWS::Logs::LogStream
+    Properties:
+      LogGroupName: !Ref AggregateChoiceDataStreamLogGroup
+      LogStreamName: s3Backup
+
+  AggregateChoiceDataStreamRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: firehose.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: firehose_delivery_policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: s3:*
+                Resource: !GetAtt AggregateChoiceDataS3Bucket.Arn
+              - Effect: Allow
+                Action: logs:PutLogEvents
+                Resource: !GetAtt AggregateChoiceDataStreamLogGroup.Arn
+
+  AggregateChoiceDataS3Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub ${Environment}-enchanted-brain-choices
```

Comment:
I'm hoping that my having deployed this template for testing doesn't cause issues with availability of this bucket name. I've deleted my test stack, so I think that we should be good to go, but may need to change this slightly if I'm wrong about that.